commit 64f201eff8702eeb08d1743bccd6dd412b3f8d8b
Author: Cole Robinson <crobinso@redhat.com>
Date:   Sat Jan 28 16:38:01 2012 -0500

    console: Fix spice scaling
    
    Actually scale, not try and resize guest resolution. We will want to add
    new UI or smarts to enable that.

diff --git a/src/virtManager/console.py b/src/virtManager/console.py
index e97704c..17b0467 100644
--- a/src/virtManager/console.py
+++ b/src/virtManager/console.py
@@ -526,10 +526,15 @@ class SpiceViewer(Viewer):
             self.spice_session.connect()
 
     def get_scaling(self):
-        return self.display.get_property("resize-guest")
+        if not has_property(self.display, "scaling"):
+            return False
+        return self.display.get_property("scaling")
 
     def set_scaling(self, scaling):
-        self.display.set_property("resize-guest", scaling)
+        if not has_property(self.display, "scaling"):
+            logging.debug("Spice version doesn't support scaling.")
+            return
+        self.display.set_property("scaling", scaling)
 
 
 class vmmConsolePages(vmmGObjectUI):
